src/utils/setLocalStorage.js

```diff
@@ -0,0 +1,8 @@
+export const setLocalStorage = (questionNum) => {
+  document.querySelectorAll('li').forEach((answer) => {
+    // saving selected answer to local storage
+    answer.addEventListener('click', (event) => {
+      localStorage.setItem(questionNum, event.target.innerText[0]);
```

Comment:
This function has to keep only `localStorage.setItem(questionNum, event.target.innerText[0]);`